apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.scss

```diff
@@ -47,7 +47,7 @@
     background-color: $white;
     z-index: 1000;
     width: 257px;
-    border: 1px solid $gainsboro;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine